From 04d7a03acc24ccec00e285d71f0c61fb3b36b72a Mon Sep 17 00:00:00 2001
From: Subrata Chatterjee <schatterjee@habana.ai>
Date: Fri, 23 Dec 2022 10:42:12 +0200
Subject: [PATCH] [SW-103768] gaudi2: sram bist: remove redundant func

This patch removes gaudi2_sram_id_sel() call after ring
specific pattern. This function is redundant as same
is called at the start of every iteration for each ring
in gaudi2_sram_self_repair().

Change-Id: I147325f69b9c5ddcb4b1a14cf35ef568ff7a45d1
Signed-off-by: Subrata Chatterjee <schatterjee@habana.ai>
---

diff --git a/arch/gaudi2/arc/asic.c b/arch/gaudi2/arc/asic.c
index 12ae783..3d0acab 100644
--- a/arch/gaudi2/arc/asic.c
+++ b/arch/gaudi2/arc/asic.c
@@ -271,7 +271,6 @@
 				preboot_puts_hex("bisr failed val: ", val, 0);
 			}
 		}
-		gaudi2_sram_id_sel();
 	}
 }
 
